apps/pie-microsite/src/assets/styles/components/_tokensTable.scss

```diff
@@ -51,6 +51,20 @@ $tokensTable-item-vertical-padding: f.spacing(e);
     }
 }
 
+.c-tokensTable-example-container--elevation {
+    background: var(--example-container);
```

Comment:
let's use `background-color` rather than `background`. Otherwise, when you use the shorthand property, you're setting other background properties that might need to be reset/undone later down the cascade. A better explanation might be found [here](https://techstacker.com/difference-between-background-and-background-color-in-css/) but essentially we should strive to only set the properties we want, and nothing more :)

Reply:
oh nice thank you! always thought it was shorthand - good to know 👍